go/service/user_handler.go

```diff
@@ -58,14 +61,22 @@ func (r *userHandler) identityChange(m libkb.MetaContext) error {
 
 func (r *userHandler) passwordChange(m libkb.MetaContext, cli gregor1.IncomingInterface, category string, item gregor.Item) error {
 	m.Debug("userHandler: %s received", category)
-
-	// If the passphrase ever changes, it's now known.
-	libkb.MaybeSavePassphraseState(m, keybase1.PassphraseState_KNOWN)
-
 	r.G().NotifyRouter.HandlePasswordChanged(m.Ctx())
```

Comment:
this UI message might need to be handled in `passphraseStateUpdate`. Otherwise UI will get a callback from here, try to load current passphrase state to update UI, but get the old state because this message arrived quicker than `user.passphrase_state`